### PR TITLE
MSI: Add deprecation message to CatalogInventory SPIs

### PR DESCRIPTION
**Pull request by lbajsarowicz**
### Description (*)
Add deprecation information to CatalogInventory SPIs

### Fixed Issues (if relevant)
Not applicable

### Manual testing scenarios (*)
Not applicable

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Review by maghamed — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @maghamed, thank you for the review.
[ENGCOM-4502](https://jira.corp.magento.com/browse/ENGCOM-4502) has been created to process this Pull Request

**Comment by VasylShvorak** [comment]
**Notice:** QA not applicable

File: app/code/Magento/CatalogInventory/Model/Spi/StockRegistryProviderInterface.php [hunk 1]
```diff
@@ -15,19 +15,25 @@
 interface StockRegistryProviderInterface
 {
     /**
+     * Get stock.
```
Comment (lbajsarowicz) [inline app/code/Magento/CatalogInventory/Model/Spi/StockRegistryProviderInterface.php:18]: Isn't that against DRY? I mean - the method name is descriptive enough.

**Pull request merged**